envbuilder.go

```diff
@@ -394,6 +396,14 @@ func Run(ctx context.Context, options Options) error {
 		})
 	}
 
+	// temp move of all ro mounts
+	tempRemountDest := filepath.Join("/", MagicDir)
```

Comment:
I wonder if we should make this `/[MagicDir]/mnt`

Reply:
Not a bad idea 👍